Modules/Modules/Home Alone at Day X/task-info.yaml

```diff
@@ -0,0 +1,45 @@
+type: output
+files:
+- name: tests/output.txt
+  visible: false
+- name: src/main.rs
+  visible: false
```

Comment:
Do we have to have this invisible?

Reply:
I want to make learners get used to working on a library, not the binary. There is not much sense in referring to `main.rs` in this task.

Reply:
Ok, I decided to make it visible in this task.